docs/blog/2018-03-07-why-we-created-the-plugin-library/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Why we created the Gatsby plugin library
+date: 03-07-2018
```

Comment:
this date is in format that gatsby doesn't recognize - https://deploy-preview-4422--gatsbyjs.netlify.com/blog/2018-03-07-why-we-created-the-plugin-library/ (invalid date) and this post gets pushed to the end of post listing

this should be `"2018-03-07"`